addon/dev-prefs.json

```diff
@@ -1,5 +1,4 @@
 {
-  "BASE_URL": "http://testpilot.dev:8000",
-  "HOSTNAME": "testpilot.dev",
-  "ALLOWED_ORIGINS": "http://testpilot.dev:8000/*,https://testpilot.firefox.com/*"
+  "toolkit.telemetry.server": "http://127.0.0.1/",
+  "extensions.@testpilot-addon.SERVER_ENVIRONMENT": "local"
```

Comment:
Yeah, it's been awhile since we looked at these settings. Really should have updated them back when [I implemented the simple env drop-down selector](https://github.com/mozilla/testpilot/commit/7339a3a5885f3f308537318fbacc293353b416ba), because none of the settings in there now do anything.